ion/bitstream.go

```diff
@@ -172,6 +172,15 @@ func (b *bitstream) Next() error {
 
 	// Parse the tag.
 	code, len := parseTag(c)
+
+	// Binary-encoded structs with len 1 are special cases of sorted field-names.
```

Comment:
The sort order for field names in this case is a bit un-intuitive. I suggest spelling it out:

```
// Structs with a length code of 1 are a special case. Their length is always encoded as a VarUInt
// and their field names appear in ascending symbol ID order.
```